vita3k/shader/src/usse_program_analyzer.cpp

```diff
@@ -53,6 +53,8 @@ std::uint8_t get_predicate(const std::uint64_t inst) {
     }
 
     // VMAD normal version, predicates only occupied two bits
+    case 0b00100:
+    case 0b00101:
     case 0b00000:
         return ((inst >> 32) & ~0xFCFFFFFF) >> 24;
```

Comment:
Correctly handle the condition here please. Its shorten and not normal predicate value

Reply:
Thanks for reminding me! Note that it is actually 2 bits long.

Reply:
How about this? https://github.com/Vita3K/Vita3K/pull/587/files#diff-87a524ae6144f7f6b50e6263d12ef5daR58

Reply:
Why not